### PR TITLE
fix processing of negative scalars during parse literals

### LINKED ISSUE #92: panic on negative int as argument
When I try to pass negative integer as argument for query field, I've got an error:
```
{
  "errors": [
    {
      "message": "syntax error: invalid value",
    }
  ]
}
```
During debug I've found that source of panic was [literals.go#L191](https://github.com/neelance/graphql-go/blob/master/internal/common/literals.go#L191).
Passing positive int processed properly.

Part of schema:
```
type User {
  ...
   photos_filtered(size: Int, page: Int, locale: String!): UserPhotos
  ...
}
```
Query: 
```
query ql {
  user {
    ...
    photos_filtered(locale: "ru", size: -1) {
    	...
    }
  }
}
```

### PR DESCRIPTION
**Pull request by shnellpavel**
resolves #92

### CODE REVIEW

**Comment by shnellpavel** [comment]
@neelance What version of golang is used in CI? 
I use 1.8 and when I try to run build and test I've got a lot of errors like this:
`graphql-go/graphql.go:12:2: use of internal package not allowed`.

If I know version I will add some tests to check negatives.

**Comment by tonyghita** [comment]
@shnellpavel I've run into this version when testing in a forked repository. 

All the code still points to `neelance/graphql-go`, and go disallows the forked version to access the `internal` package of the original version.

**Comment by shnellpavel** [comment]
@tonyghita Is there any way to run tests without rewriting all imports?

**Comment by tonyghita** [comment]
Check out this article: http://code.openark.org/blog/development/forking-golang-repositories-on-github-and-managing-the-import-path. 

TLDR; You can add your fork as a remote in the main package.

**Comment by neelance** [comment]
Thanks!

**Comment by neelance** [comment]
fyi: CI is running Go 1.8.1

**Comment by shnellpavel** [comment]
@tonyghita Thanks for help!
@neelance Thanks for approve!

**Comment by shnellpavel** [comment]
@neelance test-cases were merged (https://github.com/graphql/graphql-js/pull/891)

**Comment by neelance** [comment]
Tests updated via 073edccda088f21eb03050184190bbe1e66289eb. Thanks again!
